tests/unit/source_code/samples/functional/spark-logging.py

```diff
@@ -0,0 +1,15 @@
+# ucx[legacy-context-in-shared-clusters:+1:0:+1:14] sc is not supported on UC Shared Clusters. Rewrite it using spark
+# ucx[spark-logging-in-shared-clusters:+1:0:+1:22] Cannot set Spark log level directly from code on UC Shared Clusters. Remove the call and set the cluster spark conf 'spark.log.level' instead
+sc.setLogLevel("INFO")
+setLogLevel("WARN")
```

Comment:
```suggestion
# ucx[legacy-context-in-shared-clusters:+1:0:+1:14] sc is not supported on UC Shared Clusters. Rewrite it using spark
sc.setLogLevel("INFO")

# ucx[spark-logging-in-shared-clusters:+1:0:+1:22] Cannot set Spark log level directly from code on UC Shared Clusters. Remove the call and set the cluster spark conf 'spark.log.level' instead
setLogLevel("WARN")
```

annotated comments have to be right above the line they annotate. at the moment the framework doesn't check the line offsets, as there's a bug in notebook cell parsing.

Reply:
This one is actually correct: the first code line should emit two warnings, while the second line is a valid code (assuming that `setLogLevel` is defined as a user function somewhere).

One thing worth considering is whether we want to show both these errors or only the more specific one (`spark-logging-in-shared-clusters` in this case). Right now I feel that both errors are useful, but we can change it later